lessons/09-css-style-attr/index.html

```diff
@@ -0,0 +1,35 @@
+<html>
+    <html>
+    <head>
+        <title>
+            Style Attributes
+        </title>
+    </head>
+    <body>
+        <div style="text-align: center;">
+         <h1>Home Page</h1>
+        </div>
+      
+        <div style="text-align: right;">
+                <p style="font-style: italic;">
+                  Lorem, ipsum dolor sit amet consectetur adipisicing elit. Commodi itaque, ad
+                  quo eius tempora reiciendis suscipit porro nihil, voluptate facere, cum
+                  totam molestiae ut earum quod sapiente non amet provident.
+                </p>
+                <div style="background:blue;color:  rgb(255,255,255);">
+                <p>
+                  Lorem, ipsum dolor sit amet consectetur adipisicing elit. Commodi itaque, ad
+                  quo eius tempora reiciendis suscipit porro nihil, voluptate facere, cum
+                  totam molestiae ut earum quod sapiente non amet provident.
+                </p>
+                </div>
+              
+                <p> <div style="text-transform: uppercase;">Lorem</div>, ipsum dolor sit amet consectetur adipisicing elit. Commodi itaque, ad
```

Comment:
Here we would want to use a `<span>` to target Lorem, because `<div>` is a block element it is going to push everything else to a new line. `<span>` is inline so it will not interfere with the flow of the text

```html
<p><span style="text-transform: uppercase;">Lorem</span> ipsum dolor sit amet...</p>
```